addons/stock/wizard/stock_quantity_history.xml

```diff
@@ -1,31 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <record id="view_stock_quantity_history" model="ir.ui.view">
-        <field name="name">Inventory Report</field>
+        <field name="name">Inventory Report at Date</field>
         <field name="model">stock.quantity.history</field>
         <field name="arch" type="xml">
             <form string="Choose your date">
                 <group>
-                    <group>
-                        <field name="compute_at_date" widget="radio"/>
-                        <field name="date" attrs="{'invisible': [('compute_at_date', '=', '0')]}"/>
-                    </group>
+                    <field name="inventory_datetime"/>
                 </group>
                 <footer>
-                    <button name="open_table" string="Retrieve the Inventory Quantities" type="object" class="btn-primary"/>
+                    <button name="open_at_date" string="Confirm" type="object" class="btn-primary"/>
```

Comment:
i don't see where this view is referenced now that you removed the stock account action

Reply:
removed the declaration in stock_account